src/Services/SearchService.php

```diff
@@ -77,6 +77,11 @@ class SearchService implements SearchServiceInterface
      */
     protected $configRepository;
 
+    /**
+     * @var bool
+     */
+    protected $aliveTestResult;
```

Comment:
Is this somehow cached now? Also the typehint is kinda misleading, it should actually be `bool|null`.

Reply:
Yeah, it's a leftover from a previous attempt at fixing this. It's not really used now, but I thought it could be left this way as it might be useful in the future.

Reply:
Okay, but please update the docblock.